jest.config.js

```diff
@@ -12,9 +12,15 @@ module.exports = {
   coverageThreshold: {
     global: {
       branches: 80,
-      statements: 80,
       functions: 80,
-      lines: 80
+      lines: 80,
+      statements: 80,
+    },
+    './src/APIBasedAuth.ts': {
+      branches: 100,
+      functions: 100,
+      lines: 100,
+      statements: 100
```

Comment:
I'd say we avoid having specific coverage for files and we just up the global coverage based on what we get after these latest changes.

Reply:
Absolutely!

Reply:
if we want that we're going to have to add coverage to the LOAuth which already exists too and doesn't have 100% coverage. which adds to the scope so wasn't sure that we wanted add that globally